exasol_integration_test_docker_environment/cli/cli.py

```diff
@@ -3,4 +3,22 @@
 
 @click.group()
 def cli():
+    """
+    ITDE - Itegration Test Docker Environemnt
+
+    Create and control a docker based exasol database test setup.
+
+    Examples:
+
+        Check the health of the execution environment:
+
+            $ itde health
+
+        Spwan a itde test environment:
+
+            $ itde spawn-test-environment --environment-name test \\
```

Comment:
hm, this gets problematic, when the argument names should change, we would need to remember to update this.

Reply:
As discussed, updated the PR-Template for now.